app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str) -> None:
+    file_out = command.split(" ")[1]
+    file_in = command.split(" ")[2]
+    if file_in == file_out:
+        pass
+    else:
+        with open(file_out, "r") as file_out, open(file_in, "w") as file_in:
```

Comment:
The variable names `file_out` and `file_in` are misleading. `file_out` should refer to the output file (the file being written to), and `file_in` should refer to the input file (the file being read from). Consider swapping these variable names to better reflect their roles.